web/src/beta/components/StorytellingPageSectionItem/index.tsx

```diff
@@ -0,0 +1,84 @@
+import styled from "@emotion/styled";
+import { FC } from "react";
+
+import Icon from "../Icon";
+
+type Props = {
+  icon: string;
+  title: string;
+  index: number;
+  active?: boolean;
+  onAction?: () => void;
+  onClick?: () => void;
+};
+
+const StorytellingPageSectionItem: FC<Props> = ({
+  icon,
+  title,
+  index,
+  active,
+  onAction,
+  onClick,
+}) => {
+  return (
+    <HorizontalBox>
+      <VerticalBox>
+        <IndexText>{index}</IndexText>
```

Comment:
For all text inside of components, please use the `components/Text` component and match the size to Figma 🙏🏻